README.md

```diff
@@ -10,25 +10,26 @@ Utils for RDFJS Datasets.
 Each util function can be loaded as property from the main module or by loading only the file with the same name.
 Loading the individual files can decrease the file size after packaging the JavaScript code for the browser.
 
-### Example
+### Usage
 
 Loading the function from the main module:
 
-    const resource = require('rdf-utils-dataset').resource
- 
+```js
+import { resource } from 'rdf-utils-dataset'
+```
+
 Loading the function from the file with the function name:
 
-    const resource = require('rdf-utils-dataset/resource')
+```js
+import { resource } from 'rdf-utils-dataset/resource.js'
```

Comment:
This does not match the export. Should be default-imported or changed in `resource.js`

```suggestion
import resource from 'rdf-utils-dataset/resource.js'
```